pom.xml

```diff
@@ -205,7 +205,7 @@
                 <plugin>
                     <groupId>io.takari.maven.plugins</groupId>
                     <artifactId>takari-lifecycle-plugin</artifactId>
-                    <version>1.13.7</version>
```

Comment:
I suggest that the plugin's version should be removed from this file and updated in the parent pom instead.